### DISCUSSION TITLE
when i use coc-ltex, how should i trigger the suggestion list.

### DISCUSSION THREAD

**Discussion by cstsunfu** [valentjn/vscode-ltex#599] [Q&A]
when i use coc-ltex，and i have opening the completionenable option，how should i trigger the suggestion list.

**Comment by bguo068** [comment]
I had the same question and just recently found we can use the maping  `<leader>ac` (which is mapped to coc-codeaction using `nmap <leader>ac  <Plug>(coc-codeaction)` see [coc-nvim example config](https://github.com/neoclide/coc.nvim#example-vim-configuration)) to show a menu of suggestion.

<img src="https://user-images.githubusercontent.com/13037898/172036210-7c9fe224-d245-4dba-bc6d-010505bdc429.png" width="200" />
